lib/fbe/middleware/sqlite_store.rb

```diff
@@ -73,7 +74,13 @@ def read(key)
       t.execute('UPDATE cache SET touched_at = ?2 WHERE key = ?1;', [key, Time.now.utc.iso8601])
       t.execute('SELECT value FROM cache WHERE key = ? LIMIT 1;', [key])
     end.dig(0, 0)
-    JSON.parse(value) if value
+    return unless value
+    begin
+      JSON.parse(Zlib::Inflate.inflate(value))
+    rescue Zlib::Error => e
+      @loog.info("Failed to decompress cached value for key: #{key}, error: #{e.message}")
```

Comment:
@Yegorov maybe we should also delete this key from the DB?

Reply:
@yegor256 of course, fixed.